modules/oe_translation_cdt/src/Mapper/TranslationRequestMapper.php

```diff
@@ -175,14 +175,14 @@ protected function countVolume(int $character_count): float {
   protected function createCallbacks(): CallbackCollection {
     // Set the LANGUAGE_NOT_SPECIFIED to avoid the language suffix in the URL.
     // @see \Drupal\language\Plugin\LanguageNegotiation\LanguageNegotiationUrl::processOutbound()
-    $job_status_url = Url::fromRoute(
+    $request_status_url = Url::fromRoute(
```

Comment:
Cant belive we missed it, lets improve also in TranslationRequestMapperTest the testing of getCallbackBaseUrl

Reply:
True! I've just added some additional asserts.